From e9c7a0cfca3f90f5869c43a5fcca10771d9eacb9 Mon Sep 17 00:00:00 2001
From: milaq <micha.laqua@gmail.com>
Date: Wed, 2 Nov 2016 01:12:44 +0100
Subject: [PATCH] hammerhead: disable doze

causes issues with wifi connection going haywire on wpa group rekeying
---
 overlay/frameworks/base/core/res/res/values/config.xml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/overlay/frameworks/base/core/res/res/values/config.xml b/overlay/frameworks/base/core/res/res/values/config.xml
index 3a45000..38c0f5b 100644
--- a/overlay/frameworks/base/core/res/res/values/config.xml
+++ b/overlay/frameworks/base/core/res/res/values/config.xml
@@ -342,6 +342,6 @@
     <bool name="config_powerDecoupleInteractiveModeFromDisplay">true</bool>
 
     <!-- enable doze powersaving mode -->
-    <bool name="config_enableAutoPowerModes">true</bool>
+    <bool name="config_enableAutoPowerModes">false</bool>
 
 </resources>
-- 
2.10.1

